toolkit/sidechain/domain/src/lib.rs

```diff
@@ -655,12 +711,22 @@ pub struct EpochNonce(pub Vec<u8>);
 	MaxEncodedLen,
 	Hash,
 )]
+/// Identifies a Cardano UTxO (unspent transaction output)
+///
+/// A UTxO is uniquely identified by the hash of the transaction that produced it and its (zero-based)
+/// index in the transaction's output.
+///
+/// Standard semi-human-readable encoding of a UTxO id uses a hash sign to divide the two components:
+/// `0000000000000000000000000000000000000000000000000000000000000000#1`
```

Comment:
```suggestion
/// `0000000000000000000000000000000000000000000000000000000000000000#0`
```
Would it make sense to use 0 as the ix to emphasize it's zero-based? Or does it make it even harder to find the `#`?